### DISCUSSION TITLE
Communication between webpage and c++ App

### DISCUSSION THREAD

**Discussion by awais40** [AxisCommunications/acap-computer-vision-sdk-examples#53] [Q&A] [answered]
I was using net_http.h in SDK 3 for communicating from webpage to c++ via .cgi file, I found out it is no more supported in SDK 4. I have also checked there is a new way of communicating via Monkey server, which needs a major change in my application architecture. Is there is a way for using same .cgi way in SDK 4?

**Comment by marbali8** [comment, marked as answer]
Hello!

Thank you for getting in touch with us. net_http.h was deprecated before SDK 2, so we recommend you to stop using it. Our web-server example shows how we communicate now between application and webpage using reverse proxy. This configuration is not only not limited to Monkey, but is compatible with other servers.

**Comment by Corallo** [comment]
Moving this from issue into discussion.
